media-plugins/vdr-vaapidevice/vdr-vaapidevice-1.0.0_p20190526.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit vdr-plugin-2
+
+GIT_COMMIT="d19657bae399e79df107e316ca40922d21393f80"
+
+DESCRIPTION="VDR Plugin: A VA-API output device plugin for VDR"
+HOMEPAGE="https://github.com/pesintta/vdr-plugin-vaapidevice"
+SRC_URI="https://github.com/pesintta/vdr-plugin-vaapidevice/archive/${GIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="AGPL-3"
+SLOT="0"
+IUSE="debug"
+
+RDEPEND="
+	media-video/vdr
+	media-video/ffmpeg[vaapi,X]
+	x11-libs/libva[X]
+	x11-libs/libva-intel-driver[X]
+	x11-libs/libX11
+	x11-libs/libxcb
+	x11-libs/xcb-util
+	x11-libs/xcb-util-wm
+	x11-libs/xcb-util-keysyms
+	media-libs/alsa-lib
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	virtual/pkgconfig
+"
+
+VDR_CONFD_FILE="${FILESDIR}/confd"
```

Comment:
VDR_CONFD_FILE to define here is not needed, as files/confd exist. The vdr-plugin-2.eclass will pickup this then automaticly.
VDR_CONFD_FILE is used to overwrite this handling, see media-plugins/vdr-live for a good example.
I noticed, the description in the eclass is not exact, i will fix this later in the near future.

Reply:
@hd-brummy  Got it. Ebuild is adjusted.